### PR TITLE
Responsive header in the Content Wizard

### PR DESCRIPTION
**Pull request by ashklianko**
To avoid overcomplicated logic of wrapping/unwrapping static path part and calculating it's length on value change/resize:
1. Using flex styling and showing static part with title 
2. Removed redundant elements
3. Letting dynamic input grow as much as needed and static path shrink down to zero

@alansemenov please check if this approach is appropriate

Requires https://github.com/enonic/lib-admin-ui/pull/1802

### CODE REVIEW

**Comment by ashklianko** [comment]
Fixed merge conflicts

File: src/main/resources/assets/styles/wizard/content-wizard-panel.less [hunk 1]
```diff
@@ -62,9 +55,6 @@
     &._0-240,
     &._240-360,
     &._360-540 {
-      .wizard-header {
-        max-width: calc(100% - 105px);
-      }
```
Comment (alansemenov) [inline src/main/resources/assets/styles/wizard/content-wizard-panel.less:67]: This piece of code used to fix this issue:

image

**Review by alansemenov — CHANGES_REQUESTED** [review]
image

The icon should be vertically centred

**Comment by alansemenov** [comment]
Plenty of space here to fit the entire path, but both display name and path are truncated:

image

**Comment by ashklianko** [comment]
> Plenty of space here to fit the entire path, but both display name and path are truncated:
> 
> image

I've limited length to 660 px to be same as max length of wizard step forms

**Comment by alansemenov** [comment]
> I've limited length to 660 px to be same as max length of wizard step forms

Ah, that's why... Ok, so this one is correct then :)

**Comment by ashklianko** [comment]
Updated, also moved minimize arrrow button to a new place in a toolbar

**Review by alansemenov — APPROVED** [review]

**Pull request merged**